### PR TITLE
Consider all files with .py extension as Python

### PR DESCRIPTION
**Pull request by sbrunner**
For an unknown reason, some Python file are now considered as text/plain (mine)

### CODE REVIEW

File: c2cciutils/__init__.py [hunk 1]
```diff
@@ -577,22 +577,31 @@ def graphql(query_file: str, variables: Dict[str, Any], default: Any = None) ->
...
 
     Arguments:
         mime_type: The considered MIME type
+        extensions: The considered extensions
         ignore_patterns_re: A list of regular expressions of files that we should ignore
     """
     if mime_type is None:
         mime_type = ["text/x-python", "text/x-script.python"]
+    if extensions is None:
+        extensions = ["py"]
```
Comment (arnaud-morvan) [inline c2cciutils/__init__.py:595]: Not that dot could be considered as part of the extension, ex: https://github.com/camptocamp/c2cgeoportal/blob/master/lingua.cfg
This is more compatible with os.path.splitext

File: c2cciutils/__init__.py [hunk 2]
```diff
@@ -577,22 +577,31 @@ def graphql(query_file: str, variables: Dict[str, Any], default: Any = None) ->
...
     ignore_patterns_compiled = [re.compile(p) for p in ignore_patterns_re or []]
     result = []
 
     for filename in subprocess.check_output(["git", "ls-files"]).decode().strip().split("\n"):
-        if os.path.isfile(filename) and magic.from_file(filename, mime=True) in mime_type:
+        extension_split = filename.rsplit(".", 1)
+        extension = extension_split[1] if len(extension_split) == 2 else None
```
Comment (arnaud-morvan) [inline c2cciutils/__init__.py:601]: Why not using os.path.splitext.

**Review by arnaud-morvan — APPROVED** [review]

File: c2cciutils/__init__.py (hunk 1 above)
Comment (sbrunner) [reply c2cciutils/__init__.py:595]: +1

File: c2cciutils/__init__.py (hunk 2 above)
Comment (sbrunner) [reply c2cciutils/__init__.py:601]: +1

**Pull request merged**